fulp_modules/hardhat_nolight/lampless_hardhat.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/clothing/head/hardhat/lampless_hardhat
+	name = "lampless hardhat"
+	desc = "Believe it or not, this was actually designed by a moth. Goes well with welding goggles."
```

Comment:
Don't see why you should bring up the race of the person that designed it.

Reply:
There is a running joke on Discord with people thinking I should have picked the deviant tastes trait considering my character and the PR.